omero/sysadmins/unix/install-web/install-fastcgi.txt

```diff
@@ -1,5 +1,11 @@
-OMERO.web FastCGI deployment
-============================
+OMERO.web FastCGI deployment (DEPRECATED)
+=========================================
+
+.. note::
+    Deprecated since version 5.1.4: OMERO.web deployment using FastCGI
+    is deprecated and will be removed in OMERO 5.2.
+    Use (see :doc:`install-wsgi`)
```

Comment:
This last line reads rather strangely, I'd remove the brackets and 'see' and make it:
Use `:doc:`install-wsgi` as an alternative.`